x-pack/plugins/alerting/server/task_runner/transform_action_params.ts

```diff
@@ -143,6 +143,15 @@ export function transformSummaryActionParams({
   const variables = {
     kibanaBaseUrl,
     date: new Date().toISOString(),
+    // For backwards compatibility with security solutions rules
+    context: {
+      alerts: alerts.all.data ?? [],
+      results_link: ruleUrl,
+      rule: rule.params,
```

Comment:
I was playing with this yesterday and realised that we need to reflect what we have [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/lib/detection_engine/rule_actions_legacy/logic/notifications/schedule_notification_actions.ts#L86) and [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts#L157).

```
rule: mapKeys(snakeCase, {
        ...rule.params,
        name: rule.name,
        id: rule.id,
      }),
```

Without these changes context won't be compatible with the security solution.

Reply:
Updated in this commit: https://github.com/elastic/kibana/pull/154864/commits/fa912bf4e393583ed010e53d21b42c72de353ac9